speedy-recovery-client/src/components/secured/patientinformation/PatientInfo.js

```diff
@@ -4,12 +4,20 @@ class PatientInfo extends Component {
   render() {
     const { user } = this.props;
 
+    //TODO: fix the getDate call below
```

Comment:
Is there any particular problem with this? Commented out code is kind of a code smell. If you can't fix / implement now, maybe just remove it. If you want to come back later, you can still find it in the git history.

Reply:
So when I fetch resources of patient/practitioner, their birthdate is converted into a Date object using new Date() in the mapping function. Then to show their bday in the frontend i use getDate() to show the date from the Date object. Everything works fine in the frontend and I can show their bday too. The problem comes when I run the tests. I have formatted the birthdate in the test example to be a Date object. But seems like it returns a String date so thats why getDate() method causes an error since it can only be used with Date objects. I will try to fix it now

Reply:
Any progress? Feel free to share the code of the problematic test. Sometimes the way a `Date` object is tranformed to a string can be just a different format, even though the test logic is correct.

Reply:
I just resolved the problem. Let me know whether its good enough or not.